src/main/webapp/app/exercises/shared/exam-exercise-row-buttons/exam-exercise-row-buttons.component.html

```diff
@@ -35,6 +35,16 @@
         <span class="d-none d-md-inline" jhiTranslate="artemisApp.courseOverview.exerciseDetails.instructorActions.submissions">Submissions</span>
     </a>
 
+    <!-- Example Submission -->
+    <a
+        *ngIf="course.isAtLeastInstructor"
```

Comment:
I assume this should use the editor rule too. (As it looks the routes allow editors access when you enter the link directly anyway.)

Reply:
done.